aws/vpc-public-only/outputs.tf

```diff
@@ -0,0 +1,24 @@
+// Default Security Group ID
```

Comment:
This comment seems to have become disconnected from the code it's describing.

Reply:
you're right, thanks